src/cms/models/events/event.py

```diff
@@ -46,34 +62,77 @@ def languages(self):
 
     @property
     def is_recurring(self):
+        """
+        This property checks if the event has a recurrence rule and thereby determines, whether the event is recurring.
+
+        :return: Whether the event is recurring or not
+        :rtype: bool
+        """
         return bool(self.recurrence_rule)
 
     @property
     def is_all_day(self):
+        """
+        This property checks whether an event takes place the whole day by checking if start time in minimal and end
```

Comment:
Typo: should be "start time is minimal"

Reply:
Well spotted! It seems that you actually read all of these 1,6k lines  :laughing:

Reply:
Not at all - I just started reading it! :laughing: